website/addons/osfstorage/routes.py

```diff
@@ -112,6 +112,61 @@
             views.osfstorage_copy_hook,
             json_renderer,
         ),
+
+        Rule(
```

Comment:
No, do:
POST, DELETE,GET `/project/<pid>/osfstorage/<fid>/checkouts/`

and 

POST, DELETE,GET `/project/<pid>/osfstorage/checkouts/`